src/acceptance-test/java/org/zalando/nakadi/webservice/SettingsControllerAT.java

```diff
@@ -58,6 +59,8 @@ public void testGetFlooders() throws Exception {
         final EventType eventType = NakadiTestUtils.createEventType();
         blacklist(eventType.getName(), BlacklistService.Type.CONSUMER_ET);
 
+        Thread.sleep(TimeUnit.SECONDS.toMillis(1)); // give time for TreeCache refresh
```

Comment:
Ok, I will just remove this line and try to push "build" on travis several time.